webapi/src/main/scala/org/knora/webapi/util/ConstructResponseUtilV2.scala

```diff
@@ -441,20 +452,20 @@ object ConstructResponseUtilV2 {
     }
 
     /**
-     * This method creates a map of property IRIs to sequences of value IRIs by filtering all necessary assertions and object IRIs
+     * Converts a [[ConstructPredicateObjects]] to a map of property IRIs to sequences of value IRIs.
      *
-     * @param assertionsExplicit        assertion contains all non inferred statements
-     * @param valueObjectIris           set of all value object IRIs
-     * @return                          the sequence of IRIs
+     * @param assertionsExplicit all non-inferred statements.
+     * @param valueObjectIris    a set of all value object IRIs.
+     * @return a map of property IRIs to sequences of value IRIs.
      */
-    private def mapIriSequence(assertionsExplicit: ConstructPredicateObjects, valueObjectIris: Set[IRI] ) :  Map[SmartIri, Seq[IRI]] =  {
-        val valuePropertyToObjectIris: Map[SmartIri, Seq[IRI]] = assertionsExplicit.map {
+    private def mapIriSequence(assertionsExplicit: ConstructPredicateObjects, valueObjectIris: Set[IRI]): Map[SmartIri, Seq[IRI]] = {
```

Comment:
What about a clearer name for the method (https://github.com/dasch-swiss/knora-api/pull/1658#discussion_r442705413)?

Reply:
Done in f883c02.